src/osemgrep/cli_scan/Validate_subcommand.ml

```diff
@@ -87,16 +87,20 @@ let run (conf : conf) : Exit_code.t =
          *)
         let targets =
           rules_and_origin
-          |> Common.map_filter (fun x ->
-                 (* TODO: stricter: warn if no origin (meaning URL or registry) *)
-                 x.Rule_fetching.origin)
+          |> Common.map_filter (fun (x : Rule_fetching.rules_and_origin) ->
+                 match x.origin with
+                 | Local_file path -> Some path
+                 | Other_origin ->
+                     (* TODO: stricter: warn if no origin (meaning URL or registry) *)
+                     None)
         in
         let in_docker = !Semgrep_envvars.v.in_docker in
         let (config : Rules_config.t) =
           Rules_config.parse_config_string ~in_docker metarules_pack
         in
         let metarules_and_origin =
           Rule_fetching.rules_from_dashdash_config ~token_opt
+            ~rewrite_rule_ids:true (* default *)
```

Comment:
it's a bit confusing that sometimes ~rewrite_rule_ids is a boolean and sometimes it's this Parse_rule.origin type.